nutils/types.py

```diff
@@ -1492,6 +1492,56 @@ def del_lru(self):
       lru = self._sorted_keys.pop()
     del self[lru]
 
+def lru_cache(func=None, maxsize=None):
```

Comment:
`frozenarray.lru` used to have `maxsize=128`, so caution needs be taken while moving from `frozenarray.lru`.
commit: https://github.com/evalf/nutils/pull/619/commits/19a51cbb506649aa91b64a3604edf27cce15c79f

Reply:
Even though cache entries are expected to self-destruct with the buffer, open ended caches are probably a bad idea. I changed the default back to 128.